### DISCUSSION TITLE
Thank you for the module and awesome themes 😜❤😉😎

### DISCUSSION THREAD

**Discussion by SirTwsted** [JanDeDobbeleer/oh-my-posh#821] [General]
Thank you for making this.  and thanks for the awesome themes.
Favorite so far are:::  powerlevel10k_rainbow

Is it possible to make jandedobbeleer double lined option.?  like --> powerlevel10k_rainbow

Anyways, having lot of fun with these themes 😉😎👌

**Comment by JanDeDobbeleer** [comment]
@SirTwsted you can always create your own theme based on both. How would you like to see the two lines?

**Reply by Victor-Nyagudi** [reply]
@JanDeDobbeleer  Thought I'd jump in here too and say thanks for making Oh My Posh. 💯

It's a great tool that improves the experience while using the terminal.
